string-manipulation-1/src/is-lower-cased.js

```diff
@@ -0,0 +1,10 @@
+/* exported isLowerCased */
+function isLowerCased(word) {
+  const newWord = word.split('-').join('');
+  for (let i = 0; i < newWord.length; i++) {
+    if (newWord[i] === newWord[i].toUpperCase()) {
+      return false;
+    }
+  }
+  return true;
+}
```

Comment:
Rather than doing a loop, we can actually just check the whole string at once.
```
function isLowerCased(word) {
  return word === word.toLowerCase();
}
```

Reply:
Thank you Shawn! I definitely overthought on these two exercises.